src/variant.ts

```diff
@@ -1,27 +1,45 @@
-const { ParsingError, InputValidationError } = require('./error');
-const {
-    createPosition, createBreakRepr, convertPositionToJson, parsePosition,
-} = require('./position');
-const {
+import { ParsingError, InputValidationError } from './error';
+import {
+    createPosition, createBreakRepr, convertPositionToJson, parsePosition, AnyPosition
+} from './position';
+import {
     NOTATION_TO_TYPES,
     TYPES_TO_NOTATION,
     NONSENSE,
     TRUNCATING_FS,
-} = require('./constants');
-const { parseContinuous, getPrefix } = require('./continuous');
+    Prefix
+}  from './constants';
+import { parseContinuous, getPrefix } from './continuous';
 
 const POSITION_ATTRS = ['break1Start', 'break1End', 'break2Start', 'break2End'];
 
+type OntologyTerm = {
```

Comment:
while not quite as accurate, I think this would be better as just
```
type OntologyTerm = {
    name?: string;
    sourceId?: string;
    sourceIdVersion?: string;
    displayName?: string;
}
```
it doesn't seem like this union is gaining you much if anything

Reply:
done